.github/workflows/petwise.yml

```
@@ -1,9 +1,10 @@
 name: Petwise CI
 
-on: push
+on: [push, pull_request]
 
 jobs:
-  build:
+  push_job:
```

```suggestion
  build apk:
```
(replace space with `_` if necessary)